diagnostic.md

```diff
@@ -176,6 +227,8 @@ with the class `.stretch` be at a _screen width_ of 550px?
 }
 ```
 
-<!-- your answer starts here -->
+<!-- not sure; not enough time; but going to take a guess
+550px
+and 800px? -->
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have .stretch be 25% and at 550px width, 25% is 137.5 px.